contracts/introspection/ERC165Query.sol

```diff
@@ -0,0 +1,83 @@
+pragma solidity ^0.4.20;
+
+
+/**
+ * @title ERC165Query
+ * @dev https://github.com/ethereum/EIPs/blob/master/EIPS/eip-165.md
+ */
+library ERC165Query {
+  bytes4 constant INVALID_ID = 0xffffffff;
+  bytes4 constant ERC165_ID = 0x01ffc9a7;
+
+  /**
+   * @notice Query if a contract implements an interface
+   * @param _contract The address of the contract to query for support of an interface
+   * @param _interfaceId The interface identifier, as specified in ERC-165
+   * @return true if the contract at _contract indicates support of the interface with
+   * identifier _interfaceId, false otherwise
+   * @dev Interface identification is specified in ERC-165. This function
+   * uses less than 30,000 gas.
+   */
+  function doesContractImplementInterface (
+    address _contract, 
+    bytes4 _interfaceId
+  ) 
+    external 
+    view
+    returns (bool) 
+  {
+    uint256 success;
+    uint256 result;
+
+    (success, result) = noThrowCall(_contract, ERC165_ID);
+    if ((success==0)||(result==0)) {
```

Comment:
Please add spacing here between the `||` operator to comply with our code style.

Reply:
fixed